src/styles/search__label.css

```diff
@@ -0,0 +1,32 @@
+.search__label {
+  position: relative;
+  display: block;
+  width: 100%;
+  height: 100%;
+}
+
+.search__label::after {
+  position: absolute;
+  content: '';
+  display: block;
+}
+
+.search__label_big::after {
+  background-image: url(../images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
```

Comment:
Consider using a CSS variable for the background image URL since it's repeated in both `.search__label_big::after` and `.search__label_small::after`. This will make it easier to manage if the image needs to be changed in the future.